src/test/java/org/primeframework/mvc/StaticResourceTest.java

```diff
@@ -15,12 +15,48 @@
  */
 package org.primeframework.mvc;
 
+import org.primeframework.mvc.security.MockStaticClasspathResourceFilter;
+import org.primeframework.mvc.security.MockStaticResourceFilter;
 import org.testng.annotations.Test;
 
 /**
  * @author Brian Pontarelli
  */
 public class StaticResourceTest extends PrimeBaseTest {
+  @Test
+  public void get_classpath_resolution() {
+    // The default filter will deny all requests for class path resolution.
+    // - Unless this exists in the configured static directory, it will always be a 404.
+    simulator.test("/control-templates/button.ftl")
+             .get()
+             .assertStatusCode(404);
+
+    // Allow *.ftl resources through the filter
+    MockStaticClasspathResourceFilter.TestFunction = uri -> uri.endsWith(".ftl");
+
+    simulator.test("/control-templates/button.ftl")
+             .get()
+             .assertStatusCode(200)
+             .assertContentType("content/unknown")
+             .assertBodyContains("[#ftl/]")
+             .assertContentLength(470);
+
+    // We used to ignore .class URIs by default. We still do, but no longer explicitly. The class path resolution rejects everything by default.
+    // - Ensure we can't look this up w/out modifying the filter.
+    simulator.test("/org/primeframework/mvc/PrimeMVCRequestHandler.class")
+             .get()
+             .assertStatusCode(404);
+
+    // Allow *.class resources through the filter
+    MockStaticClasspathResourceFilter.TestFunction = uri -> uri.endsWith(".class");
+
+    // Not sure why you'd want to do this, but it is up to the user to configure the filter.
+    simulator.test("/org/primeframework/mvc/PrimeMVCRequestHandler.class")
+             .get()
+             .assertStatusCode(200)
+             .assertContentLength(3176);
```

Comment:
Hard-coding content length here seems even worse. `button.ftl` may be pretty static, but `class` files are more likely to change.

Reply:
I think that is ok. The intent is to ensure I am successfully reading this file exactly. If it breaks we can change it.